npe2/manifest/contributions/_commands.py

```diff
@@ -60,31 +61,32 @@ class CommandContribution(BaseModel):
     )
     _valid_pyname = validator("python_name", allow_reuse=True)(_validators.python_name)
 
-    # short_title: Optional[str] = Field(
-    #     None,
-    #     description="Short title by which the command is "
-    #     "represented in the UI",
-    # )
-    # category: Optional[str] = Field(
-    #     None,
-    #     description="Category string by the command is grouped in the UI",
-    # )
-    # icon: Optional[Union[str, Icon]] = Field(
-    #     None,
-    #     description=(
-    #         "Icon which is used to represent the command in the UI."
-    #         " Either a file path, an object with file paths for dark and light"
-    #         "themes, or a theme icon references, like `$(zap)`"
-    #     ),
-    # )
-    # enablement: Optional[str] = Field(
-    #     None,
-    #     description=(
-    #         "Condition which must be true to enable the command in the UI "
-    #         "(menu and keybindings). Does not prevent executing the command "
-    #         "by other means, like the `executeCommand` api."
-    #     ),
-    # )
+    short_title: Optional[str] = Field(
+        None,
+        description="Short title by which the command is represented in "
+        "the UI. Menus pick either `title` or `short_title` depending on the context "
+        "in which they show commands.",
+    )
+    category: Optional[str] = Field(
+        None,
+        description="Category string by which the command may be grouped in the UI.",
+    )
+    icon: Optional[Union[str, Icon]] = Field(
+        None,
+        description="Icon used to represent this command in the UI, on "
+        "buttons or in menus. These may be [superqt](https://github.com/napari/superqt)"
+        " fonticon keys, such as `'fa6s.arrow_down'`; though not that plugins are "
+        "expected to depend on any fonticon libraries they use, e.g "
+        "[fonticon-fontawesome6](https://github.com/tlambert03/fonticon-fontawesome6).",
+    )
+    enablement: Optional[str] = Field(
+        None,
+        description=(
+            "Condition which must be true to enable the command in the UI "
```

Comment:
```suggestion
            "Expression which must evaluate as true to enable the command in the UI "
```

Adding an example to the description might be helpful here too just to show precisely what's expected.

Reply:
I agree... but I think we need to make a full page of docs on expressions with all of the valid names that they can use.  It will look like this: https://code.visualstudio.com/api/references/when-clause-contexts